internal/crdReplicator/crdReplicator_operator_test.go

```diff
@@ -145,6 +155,9 @@ var _ = Describe("CRD Replicator Operator Tests", func() {
 		foreignCluster.Status = *statusCopy
 		Expect(cl.Status().Update(ctx, &foreignCluster)).To(Succeed())
 
+		foreignCluster.Spec.NetworkingEnabled = discoveryv1alpha1.NetworkingEnabledYes
+		Expect(cl.Update(ctx, &foreignCluster)).To(Succeed())
```

Comment:
Would add that directly to the foreign cluster definition above, since this is the default value nonetheless.